riocli/apply/__init__.py

```diff
@@ -35,11 +35,11 @@
               help='Dry run the yaml files without applying any change')
 @click.option('--show-graph', '-g', is_flag=True, default=False,
               help='Opens a mermaid.live dependency graph')
-@click.option('--values', '-v',
+@click.option('--values', '-v', multiple=True, default=(),
```

Comment:
todo: We also need this in template command.

Reply:
Done.